opensearch/src/test/java/org/opensearch/sql/opensearch/data/type/OpenSearchDataTypeTest.java

```diff
@@ -213,26 +228,23 @@ public void fields_and_properties_are_readonly() {
   @Test
   // Test and type added for coverage only
   public void of_null_MappingType() {
-    assertThrows(IllegalArgumentException.class, () -> OpenSearchDataType.of(MappingType.Invalid));
+    assertNotNull(OpenSearchDataType.of(MappingType.Invalid));
```

Comment:
I don't think this is desired.  Can we revert this change to throw an exception with an invalid type?

Reply:
Won't do as this is expected.